# Request
float32 x_pos
float32 y_pos
---
# Result
uint64 duration
---
# Feedback
float32 dist_remaining